keyboards/sck/m0116b/keymaps/default/keymap.c

```diff
@@ -0,0 +1,126 @@
+/* Copyright 2018 jrfhoutx
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define CALTDEL LCTL(LALT(KC_DEL))
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+ /* Keymap BASE: (Base Layer) Default Layer
+   * ,---------------------------------------------------------------------------------.
+   * |                     |    CALTDEL   |                                            |
+   * `---------------------------------------------------------------------------------'
+   * ,-----------------------------------------------------------. .-------------------.
+   * | ~ | 1 |  2|  3|  4|  5|  6|  7|  8|  9|  0|  -|  =|Backsp | |MO(1)| / | *  |Paus|
+   * |-----------------------------------------------------------| |-------------------|
+   * |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|     | | 7  | 8  | 9  | +  |
+   * |-----------------------------------------------------|     | |-------------------|
+   * |Ctrl   |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|Return | | 4  | 5  | 6  | -  |
+   * |-----------------------------------------------------------' |-------------------|
+   * |Shift   |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift     | | 1  | 2  | 3  | Ent|
+   * |-----------------------------------------------------------' `--------------|    |
+   * |CAPS|Opt |Cmd | ` |    Space        | \ |Left|Up|Down|Right| | 0       | .  |    |
+   * `-----------------------------------------------------------' `-------------------'
+   */
+[0] = LAYOUT_M0116( /* Base */
```

Comment:
Needs to match the layout macro name used by info.json.

Changing the macro name because QMK custom is `LAYOUT_` followed by a fully-lowercase suffix.

```suggestion
[0] = LAYOUT_m0116_ansi( /* Base */
```